lambda-code/cognito-email-sender/main.ts

```diff
@@ -68,13 +73,26 @@ export const handler: Handler = async (event) => {
         },
       });
     } catch (err) {
-      console.error(
-        JSON.stringify({
-          status: "failed",
-          message: "Notify Failed To Send the Code.",
-          error: (err as Error).message,
-        })
-      );
+      if (err instanceof AxiosError) {
+        // Error Message will be sent to slack
+        console.error(
+          JSON.stringify({
+            level: "error",
+            msg: `Failed to send password reset email to ${userEmail}`,
+            error: err.response?.data?.errors
+              ? JSON.stringify(err.response.data.errors)
+              : err.message,
+          })
+        );
+      } else {
+        console.error(
+          JSON.stringify({
+            status: "failed",
+            message: "Failed to send Notify Email.",
```

Comment:
Could we still be more specific with this message? i.e. `Failed to send password reset email`

Reply:
yup, will change to align with message above.